### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by ferrastas**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (renan) [inline .github/dependabot.yml:6]: Missing newline, as does new files

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      GO111MODULE: on
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Test
+        run: make verify
```
Comment (renan) [inline .github/workflows/verify.yml:26]: Newline

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	GO111MODULE=off go get -u golang.org/x/lint/golint
+	GO111MODULE=off go get -u golang.org/x/tools/cmd/goimports
+	GO111MODULE=off go get -u github.com/GeertJohan/fgt
+	GO111MODULE=off go get -u github.com/kisielk/errcheck
+
+go-mod:
+	go mod tidy
+	go mod verify
+
+verify: go-mod test install-tools lint
```
Comment (renan) [inline Makefile:21]: Newline

**Review by renan — CHANGES_REQUESTED** [review]
Nice to have tests and linting! Other than the missing new lines, I think it is good.

File: .github/workflows/verify.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  pull_request:
+    branches:
+      - master
+name: Verify
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
```
Comment (renan) [inline .github/workflows/verify.yml:14]: Do we actually need to test on 1.14?

Comment (ferrastas) [reply .github/workflows/verify.yml:14]: Not really, as go.mod already reflects 1.15

**Comment by ferrastas** [comment]
@renan fixed

**Review by renan — APPROVED** [review]

**Pull request merged**